src/image/loading_displaying.js

```diff
@@ -424,6 +424,14 @@ function _sAssign(sVal, iVal) {
  * to explain further:
  * <img src="assets/drawImage.png"></img>
  *
+ * This function can also be used to draw images without distorting the orginal aspect ratio,
+ * by adding 9th parameter fit which can either be COVER or CONTAIN.
```

Comment:
Super minor grammar suggestion: I think a few commas on this line and the next will make this a tad easier to read:
```diff
-  * by adding 9th parameter fit which can either be COVER or CONTAIN.
+  * by adding 9th parameter, fit, which can either be COVER or CONTAIN.
-  * CONTAIN as the name suggests contains the whole image within the specified destination box
+  * CONTAIN, as the name suggests, contains the whole image within the specified destination box
 ```